Sources/BraveWallet/Crypto/Accounts/Details/AccountDetailsView.swift

```diff
@@ -24,10 +24,13 @@ struct AccountDetailsView: View {
   @State private var isPresentingRemoveConfirmation: Bool = false
 
   @Environment(\.presentationMode) @Binding private var presentationMode
+  
+  private var isDoneDisabled: Bool {
+    name.isEmpty || !name.isValidAccountName
```

Comment:
Can we add isEmpty case inside isValidAccountName to satisfy this condition together? I believe in other place which uses isValidAccountName also does the isEmpty check.

Reply:
An empty account name is permitted when adding an account because the default account name would be used. As a result, empty string is only invalid when renaming the account.